packages/react-components/src/components/table-output-component.tsx

```diff
@@ -526,9 +549,11 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
             if (isFound) {
                 return;
             }
-
-            const currRowIndex = this.gridApi?.getLastDisplayedRow() + 1;
-            const lineIndex = await this.findNextMatchIndex(currRowIndex);
+            if (direction === Direction.NEXT) {
+                currRowIndex = this.gridApi?.getLastDisplayedRow() + 1;
```

Comment:
I'm not sure if necessary, but maybe it needs to be the first index following the last row node index that was considered by the 'current block' fix above?

And for PREVIOUS it should be the first index before the first considered row node index?

Reply:
I'll check